packages/report-server/package.json

```diff
@@ -39,6 +39,8 @@
     "body-parser": "^1.18.3",
     "cors": "^2.8.5",
     "express": "^4.16.2",
+    "lodash.pick": "^4.4.0",
+    "@types/lodash.pick": "^4.4.0",
     "mathjs": "^7.3.0",
```

Comment:
lodash doesn't packages its declaration files with its npm packages. 
If we don't expect `report-server` to be used as a library, we can put `@types/lodash.pick` to `devDependencies`.

Reference: https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html#dependencies